src/test/java/com/google/sps/AuthCheckerTest.java

```diff
@@ -0,0 +1,47 @@
+package com.google.sps;
+
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.appengine.api.users.User;
+import com.google.appengine.api.users.UserService;
+import com.google.sps.util.AuthChecker;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import static org.mockito.Mockito.*;
+
+import java.io.IOException;
+
+@RunWith(JUnit4.class)
+public class AuthCheckerTest {
+  private HttpServletResponse response;
+  private UserService userServiceNotLoggedIn;
+  private UserService userServiceLoggedIn;
+
+  @Before
+  public void setUp() throws IOException {
```

Comment:
Can you remove IOException because the Exception is never thrown in the method?

Reply:
Done.